src/util/git.ts

```diff
@@ -31,12 +34,58 @@ const didCheckoutFromMaster = async () => {
 
   if (!splitLastCheckoutMove) return FAILED_GET_LASTCHECKOUT
 
-  const isMaster = splitLastCheckoutMove[0].trim() === 'master'
+  return splitLastCheckoutMove[0].trim() === 'master'
+}
+
+const validateFiles = (changedFilesString: string, lessonId: number) => {
+  // 4 is js3 or Objects
+  if (lessonId > 4) return
+
+  const predictValidFile = (e: string) =>
+    e.includes('/')
+      ? e.split('/')[1].match(fileNameRegex)
```

Comment:
This works because of lexical scoping but this would be clearer if you created the `fileNameRegex` variable before it is used here in this this function.